### DISCUSSION TITLE
2008/07/04/%E6%80%8E%E6%A0%B7%E6%A0%B9%E6%8D%AE%E9%94%99%E8%AF%AF%E4%BB%A3%E7%A0%81%E5%BE%97%E5%88%B0%E9%94%99%E8%AF%AF%E4%BF%A1%E6%81%AF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#229] [Announcements]
# 2008/07/04/%E6%80%8E%E6%A0%B7%E6%A0%B9%E6%8D%AE%E9%94%99%E8%AF%AF%E4%BB%A3%E7%A0%81%E5%BE%97%E5%88%B0%E9%94%99%E8%AF%AF%E4%BF%A1%E6%81%AF

大多数 VI 都会带有错误处理机制，所以 VI 的前面板上会有 error in/error out 控件。如果发现有返回错误代码，之间在空间边缘处点击鼠标右键，选择 Explain Error 就可以看到详细的错误信息。

https://blog.qizhen.xyz/2008/07/04/%E6%80%8E%E6%A0%B7%E6%A0%B9%E6%8D%AE%E9%94%99%E8%AF%AF%E4%BB%A3%E7%A0%81%E5%BE%97%E5%88%B0%E9%94%99%E8%AF%AF%E4%BF%A1%E6%81%AF

**Comment by ruanqizhen** [comment]
之前博客下的留言：

arrowpig
2008年07月4日 18:15 编辑
后来不是有Err Tag了吗？当时我还想设计一套成框架的Error Handling的东东呢，后来也不了了之了
